src/sql/src/kafka_util.rs

```diff
@@ -398,3 +388,4 @@ impl ClientContext for KafkaErrCheckContext {
         MzClientContext.error(error, reason)
     }
 }
+*/
```

Comment:
I think this got straight removed in a follow-up commit in the other PR?

Reply:
ah yes, one sec!